packages/edit-site/src/components/sidebar-navigation-screen-pattern/index.js

```diff
@@ -38,11 +41,21 @@ export default function SidebarNavigationScreenPattern() {
 	return (
 		<SidebarNavigationScreen
 			actions={
-				<SidebarButton
-					onClick={ () => setCanvasMode( 'edit' ) }
-					label={ __( 'Edit' ) }
-					icon={ pencil }
-				/>
+				<>
+					<TemplateActions
+						postType={ postType }
+						postId={ postId }
+						toggleProps={ { as: SidebarButton } }
+						onRemove={ () => {
+							navigator.goTo( `/${ postType }/all` );
+						} }
```

Comment:
We should probably make the `onRemove` navigate back to the same location as the back button i.e. the category page when the user has come from there to edit the template part. The `backPath` value could be reused here.

Reply:
Thanks for picking up on this, and for the suggestion 🙇🏻